strawberryfields/ops.py

```diff
@@ -1411,6 +1413,29 @@ def _apply(self, reg, backend, **kwargs):
         return ancilla_val / s
 
 
+class PassiveChannel(Channel):
+    r"""Perform an arbitrary multimode passive operation
+
+    Args:
+        T (array): an NxN matrix acting on a N mode state
+
+    .. details::
+
+        Acts the following transformation on the state:
+
+        .. math::
+            a^{\dagger}_i \to \sum_j T_{ij} a^{\dagger}j
+    """
+
+    def __init__(self, T):
+        super().__init__([T])
+        self.ns = T.shape[0]
+
+    def _apply(self, reg, backend, **kwargs):
+        p = par_evaluate(self.p)
+        backend.passive(p[0], *reg)
```

Comment:
@jakeffbulmer I'm curious as well. Could we define a decomposition for `PassiveChannel` so that it would work on any backend? Similar to how `Interferometer` works